### ISSUE TITLE
Adding an independent command/event-listener for toggling participant pane in iFrame API.

### ISSUE THREAD

**Issue by manishagarwal99** [jitsi/jitsi-meet#9490]
### Description:

In the documentation for Jitsi API [link ](https://jitsi.github.io/handbook/docs/dev-guide/dev-guide-iframe) there are many commands and event listeners which make it possible for customizing it greatly. But the command for toggling the participant list is not there. 

When I set TOOLBAR_BUTTONS=["invite"] in interfaceConfigOverwrite, the button for toggling the participant list was visible only then (maybe because it also shows the invite option).

I checked the _external_api.js_ from [this ](https://github.com/jitsi/jitsi-meet/blob/master/modules/API/external/external_api.js) repository and there was also no event listener for this option in the _events_ property and also no _notifyShowParticipantList_ specifically for this purpose like other triggers

The main problem is that we can't toggle the participant pane using any command as we can do with the chat pane.

### Steps to reproduce:

Copy and paste the following files in VS code and use the option "Run on Live Server"

1. _index.html_

```
<html>
	<head>
		<style>
			.container {
				position: relative;
				width: 1520px;
				height: 645px;
				border: 1px red solid;
			}
			.toolbox {
				position: absolute;
				bottom: 0px;
				margin-left: 25%;
				border: 1px red solid;
				width: 50%;
				height: 50px;
			}
		</style>
		<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
		<script src="https://meet.jit.si/external_api.js"></script>
		<script src="main.js"></script>
		<script>
			$(function () {
				BindEvent();
				const urlParams = new URLSearchParams(window.location.search);
				var teacherid = urlParams.get("teacherid");
				$("#btnCreateMeetingLink").on("click", function () {
					var meeting_id = new Date().getTime();
					var murl =
						window.location.origin + "/myapp6meet.html?mid=" + meeting_id;
					$("#meetingid").attr("href", murl).text(murl);
					StartMeeting(meeting_id, teacherid);
				});
			});
		</script>
	</head>
	<body>
		<p>This will generate a meeting URL which can be used to join meeting.</p>
		<button id="btnCreateMeetingLink">Create Meeting</button>
		<a id="meetingid" href="#"></a>
		<div id="container" class="container" style="display: none">
			<div id="toolbox" class="toolbox" style="display: none">
				<button id="btnHangup">Hangup</button>
				<button id="btnCustomMic">Mic</button>
				<button id="btnCustomCamera">Camera</button>
				<button id="btnParticipantList">Participants</button>
			</div>
			<div id="jitsi-meet-conf-container"></div>
		</div>
	</body>
</html>
```

2.  _main.js_

```
var apiObj = null;
function BindEvent() {
	$("#btnHangup").on("click", function () {
		apiObj.executeCommand("hangup");
		apiObj.dispose();
	});
	$("#btnCustomMic").on("click", function () {
		apiObj.executeCommand("toggleAudio");
	});
	$("#btnCustomCamera").on("click", function () {
		apiObj.executeCommand("toggleVideo");
	});
	$("#btnParticipantList").on("click", function () {
		apiObj.executeCommand("toggleParticipantList");
	});
}

function StartMeeting(roomName, dispNme) {
	const domain = "meet.jit.si";
	const options = {
		roomName: roomName,
		width: "100%",
		height: "100%",
		parentNode: document.querySelector("#jitsi-meet-conf-container"),
		DEFAULT_REMOTE_DISPLAY_NAME: "New User",
		userInfo: {
			displayName: dispNme,
		},
		configOverwrite: {
			doNotStoreRoom: true,
			startVideoMuted: 0,
			startWithVideoMuted: true,
			startWithAudioMuted: true,
			enableWelcomePage: false,
			prejoinPageEnabled: false,
			disableRemoteMute: true,
			disableRecordAudioNotification: true,
			remoteVideoMenu: {
				disableKick: true,
			},
		},
		interfaceConfigOverwrite: {
			filmStripOnly: false,
			DEFAULT_LOGO_URL: "",
			JITSI_WATERMARK_LINK: "",
			BRAND_WATERMARK_LINK: "",
			SHOW_BRAND_WATERMARK: false,
			SHOW_JITSI_WATERMARK: false,
			SHOW_POWERED_BY: false,
			SHOW_WATERMARK_FOR_GUESTS: false,
			DEFAULT_REMOTE_DISPLAY_NAME: "New User",
			TOOLBAR_BUTTONS: [],
		},
		onload: function () {
			//alert('loaded');
			$("#joinMsg").hide();
			$("#container").show();
			$("#toolbox").show();
		},
	};
	apiObj = new JitsiMeetExternalAPI(domain, options);

	apiObj.addEventListeners({
		readyToClose: function () {
			//alert('going to close');
			$("#jitsi-meet-conf-container").empty();
			$("#toolbox").hide();
			$("#container").hide();
			$("#joinMsg").show().text("Meeting Ended");
		},
		audioMuteStatusChanged: function (data) {
			if (data.muted) $("#btnCustomMic").text("Unmute");
			else $("#btnCustomMic").text("Mute");
		},
		videoMuteStatusChanged: function (data) {
			if (data.muted) $("#btnCustomCamera").text("Start Cam");
			else $("#btnCustomCamera").text("Stop Cam");
		},
		showParticipantList: function (data) {
			console.log("MYDATA=", data);
			if (data.enabled) $("#btnParticipantList").text("Show");
			else $("#btnParticipantList").text("Hide");
		},
	});
	apiObj.executeCommand("subject", "New Room 2");
}
```
The UI looks like this :

![image](https://user-images.githubusercontent.com/68818968/124313570-a9783b00-db8e-11eb-975c-96d7c2e860ba.png)

### Expected behaviour:

On clicking the "Participants button" the participant's list should be visible and on clicking it again, it should hide. 

### Actual behaviour:

The button does not work as there is no functionality available for this.

### Server information:

- Jitsi Meet version: NA
- Operating System: NA

### Client information:

- Browser / app version: Google Chrome Version 91.0.4472.124 (Official Build) (64-bit)
- Operating System: Windows 10

### Additional information:

Maybe it is an unintended bug or I'm asking you for a new feature. But adding it to the API would be good if someone wants to see the participants list. Because in the tile view everything is congested.

**Comment by manishagarwal99** [comment]
I have fixed this in : 0b85a0157a4ac0745c6a81517b1fa3648cb367ca
But due to auto-formatting in VS Code during saving the file, a huge number of changes are seen.
You can refer to this: https://github.com/jitsi/jitsi-meet/pull/9491/files for better clarity. :)

**Comment by saghul** [comment]
I believe this was implemented already.
